scripts/uninstall.sh

```diff
@@ -255,7 +311,13 @@ askreboot(){
 	if [[ ${REPLY} =~ ^[Yy]$ ]]; then
 		printf "\\nRebooting system...\\n"
 		sleep 3
-		shutdown -r now
+
+		if [ "${PLAT}" == 'Alpine' ]
+		then
+			reboot
+		else
+			shutdown -r now
+		fi
```

Comment:
The `reboot` command exists on every OS I know, being an equivalent to `shutdown -r now`, but would be good to verify:
```suggestion
		reboot
```